### DISCUSSION TITLE
Question: WinUI 2.5 TextBox reset corner radius from rounded to 0/zero

### DISCUSSION THREAD

**Discussion by AtosNicoS** [microsoft/microsoft-ui-xaml#7481] [Q&A]
Since winui 2.5 the corner radius is set to round values. To get back the old behavior a fix is described here:
https://docs.microsoft.com/en-us/windows/uwp/design/style/rounded-corner#page-or-app-wide-cornerradius-changes

This works for all controls, except TextBoxes.

I dont know why this happens, the code looks plausibly correct to me:
https://github.com/microsoft/microsoft-ui-xaml/blob/release/2.5/dev/CommonStyles/TextBox_themeresources.xaml#L110

So maybe I am doing something wrong? I want to avoid setting the radius in each xaml again...

**Comment by StephenLPeters** [comment]
Can you try the overload location suggestion here instead: https://github.com/microsoft/microsoft-ui-xaml/blob/9052972906c8a0a1b6cb5d5c61b27d6d27cd7f11/build/NuSpecs/readme.txt#L21?

**Comment by AtosNicoS** [comment]
That works. I think it makes sense to correct the documentation then, as the suggested code does not work, but yours does.
https://docs.microsoft.com/en-us/windows/uwp/design/style/rounded-corner#page-or-app-wide-cornerradius-changes

Edit: This tutorial should be corrected as well:
https://docs.microsoft.com/en-us/windows/apps/winui/winui2/getting-started#download-and-install-the-windows-ui-library

**Comment by ranjeshj** [comment]
@chigy as FYI.
